src/app/shared/components/select/select.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="form-group" [formGroup]="group">
+  <label for="state">{{ field.label }}</label>
+  <select
+    class="custom-select"
+    [id]="field.name"
+    [formControlName]="field.name"
+    [ngClass]="{ 'error-input': isError }"
+  >
+    >
+    <option
+      *ngFor="let item of field.options; let inx = index"
+      [value]="item"
+      [selected]="inx === 0 ? item : null"
```

Comment:
instead of [selected] you can just assign the first value inside the component class

Reply:
changed it to [value]="field.value" in select